src/OmniSharp.MSBuild/SolutionParsing/ProjectBlock.cs

```diff
@@ -43,6 +43,11 @@ private ProjectBlock(string projectTypeGuid, string projectName, string relative
             Sections = sections;
         }
 
+        public bool IsAspWebsite()
+        {
+            return RelativePath.StartsWith("http://");
```

Comment:
it should probably be case insensitive. can this be `https://` too? 
btw, do you have a sample project like this? I don't think it's possible to create these types of projects in VS for quite a while now

Reply:
My project was made in Visual Studio 2013 for my work.

Steps to make one:

- Right click solution item in solution explorer
- Add
    - New website

The option is still there in Visual Studio 2017 Community Edition. Don't know if it works the same.